Automattic/vip-design-system
# Convert Wizard to Typescript

## Description

This PR was intended to have only the TS conversion of the Wizard component but, as it often happens, I tried to unravel a thread of components all connected to each other.

One of the roadblocks I've encountered was the various stories, because they compose the pages with multiple components, making it harder to finish the PR.

As such, I decided to make the Wizard.stories, a little bit simpler, by using a Form.Select instead of an autocomplete.
NOTE: Before merging this PR we should check the type change are compatible with how we're using the components

## Checklist

- [X] Test that the branch won't raise type check errors when used with other projects

## Steps to Test

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Check that the components changed show no difference in the story book.
